src/main/java/apoc/meta/Meta.java

```diff
@@ -73,6 +98,16 @@ public static Types from(String typeName) {
             return STRING;
         }
     }
+
+    private String getType(Object value) {
+        Types type = Types.of(value);
+        String typeName = type.toString();
+
+        if (value != null && value.getClass().isArray() && !typeName.contains("[]")) typeName +="[]";
```

Comment:
Cerchiamo di usare il `Type`

```suggestion
        if (!Type.NULL.equals(type) && value.getClass().isArray() && !typeName.contains("[]")) typeName +="[]";
```

Perché `&& !typeName.contains("[]")` ?

Reply:
Tutto questo pezzo possiamo spostarlo nel `Type#toString`? 
Creiamo una variabile privata `isArray` e nel `toString` facciamo qualcosa del genere:
```java
        @Override
        public String toString() {
            switch (this) {
                case LIST:
                    return "LIST? OF ANY?";
                default:
                    return super.toString() + (isArray ? "?[]" : "?");
            }
        }
```

`isArray` lo mettiamo a `true` se entra [dentro questo if](https://github.com/larusba/neo4j-apoc-procedures/blob/0a595139311303928b05fc30894a38e614674d37/src/main/java/apoc/meta/Meta.java#L72)